components/sensor/xiaomi_ble.rst

```diff
@@ -155,6 +157,13 @@ Configuration example:
           name: "CGG1 Humidity"
         battery_level:
           name: "CGG1 Battery Level"
+      - platform: xiaomi_cgg1
+        mac_address: "7A:80:8E:28:39:CD"
+        bindkey: "00112233445566778899aabbccddeeff"
```

Comment:
`bindkey` option is not (yet) supported for ´xiaomi_cgg1´

Reply:
Of course it isn't. It needs https://github.com/esphome/esphome/pull/1407 as noted in the pull request description.